lms/models/notes.py

```diff
@@ -0,0 +1,33 @@
+from flask import request
+from flask_login import current_user
+
+from lms.lmsdb.models import CommentText, Exercise, Note, User
+from lms.models.errors import ForbiddenPermission, UnprocessableRequest
+
+
+def delete():
+    note_id = int(request.args.get('noteId'))
+    note_ = Note.get_or_none(Note.id == note_id)
+    if note_.creator.id != current_user.id and note_.is_private:
+        raise ForbiddenPermission(
+            "You aren't allowed to access this page.", 403,
+        )
+    if note_ is not None:
+        note_.delete_instance()
+
+
+def create(user: User) -> Note:
+    note_text = request.args.get('note', '')
+    note_exercise = request.args.get('exercise', '')
+    privacy = request.args.get('privacy', '0')
+    if not note_text:
+        raise UnprocessableRequest('Empty notes are not allowed.', 422)
+    new_note_id = CommentText.create_comment(text=note_text).id
+
+    Note.create(
```

Comment:
`return`?

Reply:
Changed the type annotations to None.
I'm not returning anything because of the `try_or_fail` function - if there's no result - returns simple jsonify.